javascript-modules-engine/package.json

```diff
@@ -23,12 +23,12 @@
   "dependencies": {
     "@jahia/javascript-modules-library": "file:../javascript-modules-library/dist",
     "fast-text-encoding": "^1.0.6",
-    "i18next": "^23.10.1",
+    "i18next": "23.10.1",
```

Comment:
Any reasons why we fix the versions here rather than allowing minor/patch updates?

Reply:
To mark that this is the exact version we ship. Otherwise, we (or yarn) might be tempted to install a semver compatible version that would be a surprise for consumers.

Do you think this is the best way to do this?